dashboards/Business-1597334026524.json

```diff
@@ -203,6 +203,176 @@
         "alignLevel": null
       }
     },
+    {
```

Comment:
could you do a quick screen capture of the end result, just out of curiosity? 😄

Reply:
I'll make one once I have a more consistent behaviour about cancellation

Reply:
<img width="1680" alt="Screenshot 2020-08-14 at 17 18 29" src="https://user-images.githubusercontent.com/677056/90265183-7d5aab00-de52-11ea-8f20-9333f8df2d9c.png">